draft-ietf-moq-transport.md

```diff
@@ -603,8 +625,8 @@ code, as defined below:
 * Protocol Violation: The remote endpoint performed an action that was
   disallowed by the specification.
 
-* Duplicate Track Alias: The endpoint attempted to use a Track Alias
-  that was already in use.
+* Duplicate Subscription ID: The endpoint attempted to use a Publisher-Chosen
+  Subscription ID that was already in use.
```

Comment:
This is very confusing. Can there be duplicates for RSID as well as PSID. May be we need 2 errors
Duplicate PSID
Duplicate RSID 
??

Reply:
That has a different error code "Invalid Request ID" (which is missing its description here)

> Invalid Request ID: The session was closed because the endpoint used a Request ID that was smaller than or equal to a previously received request ID, or the least-significant bit of the request ID was incorrect for the endpoint.

Reply:
This is incorrect . From what i see, Subscription ID is either a PSID or RSID. This error says the error is only scoped to PSID.

Reply:
A duplicate Request ID is only an _error_ when it comes in a request message (eg: SUBSCRIBE, ANNOUNCE, etc).  Invalid Request ID is meant to capture that.  There's not really such a thing as a duplicate RSID - these only appear on the data plane, in SUBSCRIBE_DONE, and potentially UNSUBSCRIBE/SUBSCRIBE_UPDATE.  Duplicates are never an error in those places.

So this error code is only meant to capture duplicate PSIDs, we can update the name to reflect that.